contrib/rubocop.yml

```diff
@@ -0,0 +1,32 @@
+AllCops:
+  NewCops: disable
```

Comment:
I used the one that was presented in most of rubocop files.

Reply:
enabled NewCops.

Reply:
I've come to realize that implementing this change will require several steps. First, we'll need to enable it in each gem individually before enabling it globally. What are your thoughts on this approach? I'm concerned that if we don't handle it this way, we might run into conflicts or integration issues by the time the changes are merged.

Reply:
If the enabling new cops is going to lead to a lot of scope creep in this current PR, I think saving it for a followup makes sense. Since the individual gems were fairly inconsistent on their setting, it makes sense to ask the question do we really want this enabled globally? If anyone has opinions, please weigh in.